src/main/java/com/uber/cadence/samples/hello/HelloActivity.java

```diff
@@ -81,7 +83,7 @@ public static void main(String[] args) {
     // ClientOptions.newBuilder().setRpcTimeout(5 * 1000).build();
     WorkflowClient workflowClient =
         WorkflowClient.newInstance(
-            new WorkflowServiceTChannel(ClientOptions.defaultInstance()),
+            new Thrift2ProtoAdapter(IGrpcServiceStubs.newInstance()),
```

Comment:
We either need to rename Thrift2ProtoAdapter class or add wrapper class for this with naming like WorkflowServiceGrpc.